dcp_prototype/backend/browser/code/common/authorizer.py

```diff
@@ -0,0 +1,95 @@
+import os
+from functools import lru_cache
+
+import requests
+from chalice import UnauthorizedError
+from jose import jwt, JWTError
+
+USERINFO_ENDPOINT = "https://czi-single-cell.auth0.com/userinfo"
+
+AUTH0_DOMAIN = "czi-single-cell.auth0.com"
+API_AUDIENCE = "https://browser-api.dev.single-cell.czi.technology"
```

Comment:
Shouldn't the environment be parameterized?

Reply:
will do